src/commands.h

```diff
@@ -78,13 +78,19 @@ namespace commands {
 
 	inline void getConfig()
 	{
-		MSG_PRINT(FPSTR(TXT_MS)); MSG_PRINT(FPSTR(TXT_EQ));
+		MSG_PRINT(F("MS="));
 		MSG_PRINT(musterDec.MSenabled, DEC);
-		MSG_PRINT(FPSTR(TXT_FSEP)); MSG_PRINT(FPSTR(TXT_MU)); MSG_PRINT(FPSTR(TXT_EQ));
+		MSG_PRINT(F(";MU="));
 		MSG_PRINT(musterDec.MUenabled, DEC);
-		MSG_PRINT(FPSTR(TXT_FSEP)); MSG_PRINT(FPSTR(TXT_MC)); MSG_PRINT(FPSTR(TXT_EQ));
+		MSG_PRINT(F(";MC="));
 		MSG_PRINT(musterDec.MCenabled, DEC);
-		MSG_PRINT(FPSTR(TXT_FSEP)); MSG_PRINT("Mred"); MSG_PRINT(FPSTR(TXT_EQ));
+		#ifdef CMP_CC1101
+			if (cc1101::ccmode != 3) { // ASK/OOK = 3 (default)
+			MSG_PRINT(F(";MN=1;AFC="));
+			MSG_PRINT(AfcEnabled, DEC);
```

Comment:
Warum braucht es den den AFCEnable Wert bei jeder Nachricht. Ändert sich dieser?

Reply:
Die Ausgabe kommt nur bei get config und sieht dort so aus:

`config          MS=1;MU=1;MC=1;MN=1;AFC=1;Mred=1`

Ob ccmode sinnvoll wäre, kann ich jetzt nicht beurteilen. Da kommen ja dann verschiedene Rückgaben, je nachdem ob 2-FSK, 4-FSK oder GFSK eingestellt ist. Ich denke, das führt in die Irre.

Reply:
Ja, das passt, ich war gedanklich irgendwo anders als ich das Kommentar schrieb.